### ISSUE TITLE
a11y: Donation form required fields

### ISSUE THREAD

**Issue by ramiy** [WordImpress/Give#1178]
## Issue Overview

We need to use `aria-required="true"` on required fields.

![required](https://cloud.githubusercontent.com/assets/576623/19838069/daa94206-9ecf-11e6-802d-02d47320d4d3.png)
## Current Behavior

If the field is required, the HTML is `<input required>`.
## Expected Behavior

The HTML should be`<input required aria-required="true">` for screen-readers.
## Addons

We should also update all our addons - Mainly the "Form Field Manager". To confirm they add ARIA tags in the front-end.
